skopt/__init__.py

```diff
@@ -1,2 +1,3 @@
+from .gp_opt import acquisition
```

Comment:
Not sure what is best yet regarding the API, but I would rather not expose `gp_opt.acquisition` at the level of the package. `gp_opt.acquisition` is currently specific to GPs. 

For the example, I would rather do `from skopt.gp_opt import acquisition`